src/Core/ConfigurationDefaults.re

```diff
@@ -44,8 +44,8 @@ let getDefaultConfigString = configName =>
       {|
 {
     "bindings": [
-        { "key": "<C-P>", "command": "quickOpen.open", "when": [["editorTextFocus"]] },
-        { "key": "<C-TAB>", "command": "quickOpen.openFiles", "when": [["editorTextFocus"]] },
+        { "key": "<C-P>", "command": "quickOpen.workspaceFiles", "when": [["editorTextFocus"]] },
+        { "key": "<C-TAB>", "command": "quickOpen.buffers", "when": [["editorTextFocus"]] },
         { "key": "<C-V>", "command": "editor.action.clipboardPasteAction", "when": [["insertMode"]] },
         { "key": "<D-V>", "command": "editor.action.clipboardPasteAction", "when": [["insertMode"]] },
         { "key": "<D-P>", "command": "quickOpen.open", "when": [["editorTextFocus"]] },
```

Comment:
This line needs updating to `quickOpen.workspaceFiles` as well.
We have CMD-P bound on Mac, the same as VSCode.

Reply:
Thanks! Fixed locally.